vsintegration/src/FSharp.Editor/Common/CancellableTasks.fs

```diff
@@ -1096,6 +1095,13 @@ module CancellableTasks =
                 return! Task.WhenAll (seq { for task in tasks do yield start ct task })
             }
 
+        let inline waitForAll (tasks: CancellableTask<unit> seq) =
```

Comment:
There might a better name for this, open for suggestions.

Reply:
WaitAll, following TPL's WaitAll() ?
https://learn.microsoft.com/en-us/dotnet/api/system.threading.tasks.task.waitall?view=net-7.0